src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -89,6 +139,25 @@ export const HookPasswordValidatedInput = ({
   );
 };
 
+const getValidationState = ({
```

Comment:
I mentioned this in another PR but this is just a normal function, so it should just have normal arguments (not a Props object). A good rule of thumb is to minimize the number of arguments a function takes, but 2 is probably fine, I don't think we need to pack them in an object.